File: src/main/java/no/nav/service/pensjon/regler/to/BeslutningsstotteResponse.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.List;


import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning2011.InntektsberegningsResultat;
import no.nav.domain.pensjon.regler.vedtak.AbstraktBeregningsvilkar;
import no.nav.domain.pensjon.regler.vedtak.InntektEtterUforhet;
import no.nav.domain.pensjon.regler.vedtak.InntektForUforhet;
import no.nav.domain.pensjon.regler.vedtak.Uforegrad;
import no.nav.domain.pensjon.regler.vedtak.Uforetidspunkt;

/**
 * @author Magnus Bakken (Accenture), PK-9491
 * @author Lars Hartviksen (Decisive), PK-7048
 */
public class BeslutningsstotteResponse extends ServiceResponse {


    private List<AbstraktBeregningsvilkar> beregningsvilkarListe;

    private Pakkseddel pakkseddel;

    /** Resultat av beregning av forventet inntekt */
    private InntektsberegningsResultat inntektsberegningsResultat;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeslutningsstotteResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.beregning2011.InntektsberegningsResultat
import no.nav.pensjon.regler.domain.vedtak.AbstraktBeregningsvilkar


class BeslutningsstotteResponse : ServiceResponse() {
    var beregningsvilkarListe: List<AbstraktBeregningsvilkar> = ArrayList()

    /** Resultat av beregning av forventet inntekt  */
    var inntektsberegningsResultat: InntektsberegningsResultat? = null

}